airgap/terraform/user-data-scripts/script.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# (7) create nginx site configuration to be as proxy for the registry container and reload Nginx configuration
```

Comment:
Why separate to two scripts here instead of having one?

Reply:
Moved all the setup code from provision script to script.sh